core/services/bulletprooftxmanager/eth_broadcaster_test.go

```diff
@@ -34,13 +34,12 @@ import (
 func TestEthBroadcaster_ProcessUnstartedEthTxs_Success(t *testing.T) {
 	db := pgtest.NewGormDB(t)
 	ethKeyStore := cltest.NewKeyStore(t, db).Eth()
-	key, fromAddress := cltest.MustAddRandomKeyToKeystore(t, ethKeyStore, 0)
-	ethKeyStore.Unlock(cltest.Password)
+	key, fromAddress := cltest.MustInsertRandomKey(t, db, ethKeyStore, 0)
```

Comment:
Does the keystore no longer need to be unlocked

Reply:
correct, it unlocks with the default password automatically in `cltest.NewKeyStore()`